app/screens/settings/settings.js

```diff
@@ -153,6 +155,22 @@ class Settings extends PureComponent {
         Linking.openURL(config.HelpLink.toLowerCase());
     };
 
+    clearOfflineCache = () => {
+        const {actions, intl} = this.props;
+
+        Alert.alert(
+            intl.formatMessage({id: 'sidebar_right_menu.clear', defaultMessage: 'Clear Offline Store'}),
```

Comment:
I think this should have the `mobile` prefix as there isn't anything like this in the webapp